docs/workflow/using-dotnet-cli.md

```diff
@@ -70,10 +70,10 @@ Please run `dotnet new console` in the app folder and update the created `.cspro
 **You have to set the correct values for `RuntimeIdentifier` (RI) and `RuntimeFrameworkVersion`.**
 
 You can generally figure that out by looking at the packages you found in your output.
-In our example you will see there is a package with the name `Microsoft.NETCore.App.Runtime.win-x64.5.0.0-dev.nupkg`
+In our example you will see there is a package with the name `Microsoft.NETCore.App.Runtime.win-x64.6.0.0-dev.nupkg`
 
 ```
-Microsoft.NETCore.App.Runtime.win-x64.5.0.0-dev.nupkg
+Microsoft.NETCore.App.Runtime.win-x64.6.0.0-dev.nupkg
```

Comment:
Maybe worth mentioning:

When I am working with these private `-dev.pkg` packages, I always restore them into a local directory by either adding `--packages localpkgs` argument to the `dotnet publish` command line; or by using `globalPackagesFolder` in nuget.config.

Once these `-dev.nupkg` packages get into your global nuget cache, it is very easy to accidentally pickup stale version instead of the one you have built locally.

Reply:
I do not see `--packages` on dotnet publish. Do you mean `dotnet build --source ..` ?

Reply:
`--packages` is omnipresent option. All SDK commands support it.

Reply:
I see it does. Should it be listed in `dotnet --help` and in https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet ? I cannot find it.

Reply:
It is listed in `dotnet restore`: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore .

Reply:
I see. I updated instructions to use globalPackagesFolder.

Doing it this way, can I eliminate all the shenanigans with changing the minor build number? The procedure for iterating on a package is simply to delete the folder you are using for globalPackagesFolder, then publish again. Much simpler.

Reply:
Yes, that's my workflow.